### ISSUE TITLE
detect as unused imports subsumed by other imports

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#345]
In GitLab by @graingert on Feb 18, 2020, 01:03

*Please describe how you installed Flake8*

```
pip install flake8==3.7.9
```

```python
import sqlalchemy  # should be detected as unused
import sqlalchemy.exc


try:
    pass
except sqlalchemy.exc.DatabaseError:
    pass
```

**Comment by asottile** [comment]
In GitLab by @graingert on Feb 18, 2020, 01:04

changed the description

**Comment by asottile** [comment]
In GitLab by @graingert on Feb 18, 2020, 01:17

moved here https://github.com/PyCQA/pyflakes/issues/513

**Comment by asottile** [comment]
In GitLab by @graingert on Feb 18, 2020, 01:17

closed
